static/docs/user-guide/external-data/local.md

```diff
@@ -0,0 +1,311 @@
+# External Data on a Local Filesystem
+
+Sometimes we would like to keep the data and cache outside the project
+directory, so that we can search easily the code of the project.
+
+Keeping the data outside of the project, on a fixed absolute path, is also
+useful in case that there are several projects that use the same data.
+
+Another case when this would be desirable is if we want to keep the data on an
+[external drive](https://whatis.techtarget.com/definition/external-hard-drive)
+because the data is too big to fit on our home directory, or for some other
+reasons.
+
+A similar case is also when the data is on a
+[network-attached storage (NAS)](https://searchstorage.techtarget.com/definition/network-attached-storage)
+and is mounted through NFS, SSHFS, Samba, etc.
+
+## Tracking External Data
```

Comment:
It should be using the same structure - dependencies, outputs. Or other sections should be changed.

Reply:
> It should be using the same structure

It is not possible to use the same structure because the "local filesystem" is different from the other types of external data.

Reply:
why is it different?

Reply:
1. With "local" you can have an external cache (used instead of `.dvc/cache/`), with the other types this is not possible.
2. "local" is available to all the users, the other types might not be available or usually is available only one.